src/Generation/ResourcesGenerator.php

```diff
@@ -210,7 +210,7 @@ public static function generateRestConfig(ServiceDetails $serviceDetails, Servic
     private static function compileRestConfigInterfaces(ServiceDetails $serviceDetails, ServiceYamlConfig $serviceYamlConfig)
     {
         return $serviceDetails->methods
-            ->filter(fn ($method) => !is_null($method->httpRule) && !$method->isStreaming() && !$method->isMixin())
+            ->filter(fn ($method) => !is_null($method->httpRule) && ($method->isServerStreaming() || !$method->isStreaming()) && !$method->isMixin())
```

Comment:
Why is this necessary, when `$this->methodType === static::SERVER_STREAMING` is included in the function `isStreaming`?

Reply:
We _want_ to enable rest config generation `SERVER_STREAMING` but not `CLIENT_STREAMING` or `BIDIRECTIONAL_STREAMING` which `isStreaming` returns `true` for.

Reply:
Oops sorry, I missed the good ol `!` at the start of `$this->isStreaming()`